extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/HealthBuildTimeConfig.java

```diff
@@ -10,4 +10,10 @@
      */
     @ConfigItem(name = "extensions.enabled", defaultValue = "true")
     public boolean extensionsEnabled;
+
+    /**
+     * Whether or not to include the Liveness and Readiness Health endpoints in the generated OpenAPI document
+     */
+    @ConfigItem(name = "openapi.enabled", defaultValue = "false")
```

Comment:
Not a deal breaker, but wondering if there's a better name?

Maybe "openapi.included"?

To me, "openapi.enabled" doesn't really embody what happens

Reply:
Changed